VAEntrypointFEI not defined

Getting this error when compiling.
here is versions of everything

LIBDRM_SRC_NAME="libdrm-2.4.81"
LIBVA_SRC_NAME="libva-1.8.2"
LIBVAUTILS_SRC_NAME="libva-utils-1.8.2"
LIBVA_INTER_DRIVER_SRC_NAME="intel-vaapi-driver-1.8.2"


```
make[2]: Entering directory '/media/work/home/jay/git/01org/libyami/vaapi'
  CXX      libyami_vaapi_la-vaapicontext.lo
In file included from vaapicontext.cpp:27:0:
../vaapi/vaapistreamable.h: In function 'std::ostream& operator<<(std::ostream&, const VAEntrypoint&)':
../vaapi/vaapistreamable.h:108:10: error: 'VAEntrypointFEI' was not declared in this scope
     case VAEntrypointFEI:
          ^
Makefile:591: recipe for target 'libyami_vaapi_la-vaapicontext.lo' failed
make[2]: *** [libyami_vaapi_la-vaapicontext.lo] Error 1
make[2]: Leaving directory '/media/work/home/jay/git/01org/libyami/vaapi'
Makefile:669: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/media/work/home/jay/git/01org/libyami'
Makefile:495: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/vaapi/vaapistreamable.h
+++ b/vaapi/vaapistreamable.h
@@ -105,8 +105,10 @@ operator<<(std::ostream& os, const VAEntrypoint& entrypoint)
         return os << "VAEntrypointEncSliceLP";
     case VAEntrypointEncPicture:
         return os << "VAEntrypointEncPicture";
+#if VA_CHECK_VERSION(1, 0, 0)
     case VAEntrypointFEI:
         return os << "VAEntrypointFEI";
+#endif
     default:
         return os << "Unknown VAEntrypoint: " << static_cast<int>(entrypoint);
     }

